driver/pace/driver/run.py

```diff
@@ -533,6 +624,16 @@ def _step_physics(self, timestep: float):
             dt=float(timestep),
         )
 
+    def write_performance_json_output(self):
+        self.performance_config.write_out_performance(
+            self.comm,
+            self.config.stencil_config.backend,
+            self.config.dt_atmos,
+            self.performance_config.total_timer,
+            self.performance_config.times_per_step,
+            self.performance_config.hits_per_step,
```

Comment:
Similarly you could use `self` attributes for things existing on `performance_config` here. It's reasonable that if you write performance from the object it should write the performance it's tracking.

Reply:
Done for both. 
I originally had `*_per_step` as properties, so had to keep track of them from within driver class. After Tobias' suggestion, I moved them to be performance config class attributes, and things are definitely way cleaner.